Simplenote/SPAppDelegate.m

```diff
@@ -319,6 +322,8 @@ - (void)logoutAndReset:(id)sender
                 [self.simperium authenticateIfNecessary];
                 self.bSigningUserOut = NO;
 			}];
+
+            [[[NSUserDefaults alloc] initWithSuiteName:[SimplenoteConstants sharedGroupDomain]] setValue:false forKey:@"accountIsLoggedIn"];
```

Comment:
This should be `syncWidgetDefaults` instead right?

Reply:
Actually:`syncWidgetDefaults` should be invoked in three spots?

- applicationDidEnterBackground
- simperiumDidLogin **(it's in SPAppDelegate+Extensions!)**
- simperiumDidLogout **(same as above)**